opennlp-tools/src/main/java/opennlp/tools/ml/maxent/GISTrainer.java

```diff
@@ -227,7 +227,7 @@ public GISModel trainModel(ObjectStream<Event> eventStream, int iterations, int
     Map<String, String> params = new HashMap<>();
     params.put(GIS.ITERATIONS_PARAM, Integer.toString(iterations));
     params.put(GIS.CUTOFF_PARAM, Integer.toString(cutoff));
-    indexer.init(params, new HashMap<>());
+    indexer.init(params);
```

Comment:
The indexer should get access to the report map, so it can add counters to it or certain configuration settings.